### ISSUE TITLE
torch.functional.interpolate to quietly ignore "align_corners" when mode is set to "nearest"

### ISSUE THREAD

**Issue by torridgristle** [pytorch/pytorch#61182]
The feature I'd like is for torch.functional.interpolate to quietly ignore the presence of "align_corners" when it's set to "nearest" which doesn't use the "align_corners" settings.

The motivation behind this is that it is a hassle going back and removing it and putting it back in when I'm comparing bilinear and nearest interpolation in a network. If it does nothing then I don't want to delete it and retype it later. In the docs just mention it doesn't do anything when the mode is set to "nearest".

**Comment by albanD** [comment]
Is this a duplicate of https://github.com/pytorch/pytorch/issues/34808 ?

**Comment by torridgristle** [comment]
> Is this a duplicate of #34808 ?

No, it's not an issue of the output being misaligned, it's an issue of having to delete the `align_corners` argument from the function if you want to change the mode to 'nearest'.
